src/styles/global.css

```
@@ -2,6 +2,89 @@
 @plugin "daisyui";
 
 @custom-variant dark (&:where(.dark, .dark *));
+
+/* DaisyUI built-in theme customization (brand primary) */
+@plugin "daisyui/theme" {
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@plugin`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)